tests/blackbox/stratisd_cert.py

```diff
@@ -20,12 +20,44 @@
 import time
 import unittest
 
-from testlib.utils import exec_command, exec_test_command, process_exists
+import dbus
+
 from testlib.dbus import StratisDbus, clean_up
+from testlib.utils import exec_command, exec_test_command, process_exists, p_n, fs_n
 
 DISKS = []
 
 
+def make_test_pool(pool_name):
+    """
+    Create a test pool that will later get destroyed
+    :return: Object path of the created pool or None
+    """
+    pool_disk = ["/dev/nvme0n1p1"]
```

Comment:
The line above will not work on the blackbox tests under general conditions. You will have to get this device from which to construct the test pool from the devices passed as arguments to the script.